src/Skoruba.IdentityServer4.Admin.Common/Settings/ILoggingSettings.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Skoruba.IdentityServer4.Admin.Common.Settings
+{
+    /// Serilog logging settings
+    public interface ILoggingSettings
+    {
+        /// File sink
+        string File_LogPath { get; }
```

Comment:
Is it necessary to use this name convention with underscore?

Reply:
I dot not like such convention, but i don't have better idea how to highlight sink name in property name.